flask_log_request_id/extras/celery.py

```diff
@@ -24,6 +24,9 @@ def apply_async(self, *args, **kwargs):
 
         return super(RequestIDAwareTask, self).apply_async(*args, **kwargs)
 
+def add_request_id_header(headers=None, **kwargs):
+    if _CELERY_X_HEADER not in headers:
```

Comment:
In the begining I was sceptical if `headers` is actually `None` that this code will not work. By looking on Celery's source code I see that headers is always populated no need for type checking. I propose to remove default value `None` of `headers` arguments to align the semantic of the function with its singature. I also like that you do not update the headers they are already updated, this makes the function indepodent and for signals this is good practice (handle cases of double trigger).

It would be nice to also have a log entry as done in `RequestIDAwareTask.apply_async`. And don't forget a brief decsription and type hinting for `headers` argument :)